lib/src/header_parser/sub_parsers/functiondecl_parser.dart

```diff
@@ -80,6 +80,7 @@ Func? parseFunctionDeclaration(clang_types.CXCursor cursor) {
       exposeFunctionTypedefs:
           config.exposeFunctionTypedefs.shouldInclude(funcName),
       isLeaf: config.leafFunctions.shouldInclude(funcName),
+      ffiNativeConfig: config.ffiNativeConfig,
```

Comment:
Is it a possibility that we may have some functions which are ffi native and some which are looked up via a dynamic library later?

If ffinative is a global config(applied to all functions regardless) then maybe we can just keep this boolean at Library level (instead of each Func)

Reply:
That's what I did at first, but then the `lookUpBindings` can contain `Func` and `Global`. And `Global`s are not supported.
Maybe it should error on `Global`s when `FfiNative`s are wanted.

The `Func.toBindingString()` has no access to the containing `Library`, so it doesn't know the config of that library. Should we pass it in as an argument to that function? Or should we keep `Binding` fully self contained (meaning easier to unit test). Any preference?

Reply:
> Maybe it should error on Globals when FfiNatives are wanted.

It sounds better, we can also just skip generating globals and log a warning (Not supported with ffiNative)

> The Func.toBindingString() has no access to the containing Library

We can propagate ffiNative from library to Writer which is available.

Reply:
> We can propagate ffiNative from library to Writer which is available.

That kind of feels like the wrong thing, the writer just knows about writing, not about config. If `Func` contains everything that it needs for `toBindingString` that keeps the information more local. So I'll keep it in `Func`.

> It sounds better, we can also just skip generating globals and log a warning (Not supported with ffiNative)

Done.